.pipelines/restore-acquiring_code.cmd

```diff
@@ -0,0 +1,89 @@
+@echo off
+
+REM Prerequisities
+echo.
+echo Deleteing python from path...
+set PYTHONPATH=
+set PYTHONHOME=
+
+echo.
+echo Installing Visual Stuio 2019 workloads...
+c:
+mkdir c:\Downloads
+cd c:\Downloads
+curl -o vs_enterprise.exe -L "https://aka.ms/vs/16/release/vs_enterprise.exe"
+call C:\Downloads\vs_enterprise.exe modify --installPath "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise" --add Microsoft.VisualStudio.Component.VC.ATLMFC;includeRecommended --quiet --wait
+
+echo.
+echo Installing Windows Debugging Tools...
+curl -o winsdksetup.exe -L "https://go.microsoft.com/fwlink/p/?linkid=2083338&clcid=0x409"
+winsdksetup.exe /features OptionId.WindowsDesktopDebuggers /quiet
+
+REM Getting depot_tools
+echo.
+echo Cloneing the depot_tools repo in the root of the c:drive...
+c:
+cd \
+git clone https://chromium.googlesource.com/chromium/tools/depot_tools
+
+echo.
+echo Switching to the m80 branch of depot_tools...
+cd depot_tools
+git checkout chrome/3987
+git fetch
+
+echo.
+echo Downloading gclient...
+echo N | gclient
```

Comment:
Can you add a comment explaining why you're passing "N" to gclient?
We remember the reason now (because the most recent version of depot_tools dropped compatibility with python 2.7 that is currently required for building for Windows by some components), we might forget in the future.